### PR TITLE
Quick update for sizing

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/26918

### CODE REVIEW

File: install/cluster_size.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ Each {product-title} cluster is unique and the following guidelines provide samp
 * Memory reservations and limits, which establish memory capacity on a worker node to assign to a container. 
 * Persistent data, which is managed by the product and stored in the etcd cluster that is used by Kubernetes. 
 
-*Best practice:* For {ocp-short}, distribute the master nodes of the cluster across three (3) availability zones.
+*Important:* For {ocp-short}, distribute the master nodes of the cluster across three (3) availability zones.
```
Comment (cadawson) [inline install/cluster_size.adoc]: I think you can use either three or 3. You don't need both.

**Review by cadawson — APPROVED** [review]
Suggestion included. Otherwise, looks good. Thanks!

File: install/cluster_size.adoc (hunk 1 above)
Comment (cadawson) [reply install/cluster_size.adoc]: You might want to pass this by install squad. 

Not the three suggestion. The PR.

**Comment by dockerymick** [comment]
@cadawson thanks for your feedback and approval. I reached out to Jakob to request review. I will assign him after he agrees.

**Review by JakobGray — APPROVED** [review]

**Pull request merged**